language/move-prover/spec-lang/src/env.rs

```diff
@@ -1475,6 +1509,12 @@ impl<'env> ModuleEnv<'env> {
         module_env.into_function(FunId::new(self.env.symbol_pool.make(view.name().as_str())))
     }
 
+    /// Gets a FunctionEnv by FunctionInstantiationIndex.
+    /// The returned function might be from this or another module.
+    pub fn get_function_by_inst_idx(&self, idx: FunctionInstantiationIndex) -> FunctionEnv<'_> {
```

Comment:
This function does not need to be public, and perhaps consider unfolding it directly at the caller side.

We very rarely need to work with the handles, since the original bytecode will be translated to our own, which is not longer handle based.

Reply:
I unfolded the function at the caller side, and removed this function.